### PR TITLE
Get tickmap

### PR DESCRIPTION
**Pull request by Sniezka1927**

### CODE REVIEW

File: test/sdk/e2e/get-tickmap.test.ts [hunk 1]
```diff
@@ -0,0 +1,225 @@
+import { ONE_ALPH, sleep, web3 } from '@alephium/web3'
```
Comment (DevRozaDev) [inline test/sdk/e2e/get-tickmap.test.ts:1]: unused import

File: test/sdk/e2e/get-tickmap.test.ts [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+describe('init invariant test', () => {
+  const initialFee = 0n
+  const [fee] = getBasicFeeTickSpacing()
+  const tickSpacing = 1n
+  const initSqrtPrice = 10n ** 24n
+  const supply = 10n ** 10n
+  const lowerTickIndex = GlobalMinTick
+  const upperTickIndex = GlobalMaxTick
+  const ticks = [-221818n, -221817n, -58n, 5n, 221817n, 221818n]
```
Comment (DevRozaDev) [inline test/sdk/e2e/get-tickmap.test.ts:33]: it would better show intent by setting the first and last tick GlobalMinTick and GlobalMaxTick respectively

**Review by DevRozaDev — APPROVED** [review]

**Review by DevRozaDev — APPROVED** [review]

File: src/consts.ts [hunk 3]
```diff
@@ -26,6 +28,7 @@ export const MaxU256 =
 
 export const MaxSqrtPrice = 65535383934512647000000000000n
 export const MinSqrtPrice = 15258932000000000000n
+export const MAX_BATCHES_QUERIED = 18n
```
Comment (wojciech-cichocki) [inline src/consts.ts:31]: What is the reason for this batch limit? Is it due to computations or response size?

Comment (Sniezka1927) [reply src/consts.ts:31]: The limit is due computations. Querying 18 batches required 4.98m out of 5m gas limit. Every tickmap with tick spacing greater than one can be retrieved within single query.

The alephium queries has also default http response size limit, that parameter shouldn't cause problems in our implementation.

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**